database/DatabaseExample.xcodeproj/project.pbxproj

```diff
@@ -267,6 +273,13 @@
 			name = ViewControllers;
 			sourceTree = "<group>";
 		};
+		244A75058F265EBB9D457A95 /* Pods */ = {
```

Comment:
Remove Pods references in this file.  This can be done manually or from the Xcode UI